website/www/site/content/en/documentation/dsls/sql/walkthrough.md

```diff
@@ -20,18 +20,15 @@ limitations under the License.
 
 This page illustrates the usage of Beam SQL with example code.
 
-## Row
+## Beam Schemas and Rows
 
-Before applying a SQL query to a `PCollection`, the data in the collection must
-be in `Row` format. A `Row` represents a single, immutable record in a Beam SQL
-`PCollection`. The names and types of the fields/columns in the row are defined
-by its associated [Schema](https://beam.apache.org/releases/javadoc/{{< param release_latest >}}/index.html?org/apache/beam/sdk/schemas/Schema.html).
-You can use the [Schema.builder()](https://beam.apache.org/releases/javadoc/{{< param release_latest >}}/index.html?org/apache/beam/sdk/schemas/Schema.html) to create
-`Schemas`. See [Data
-Types](/documentation/dsls/sql/data-types) for more details on supported primitive data types.
+A SQL query can only be applied to a a `PCollection<T>`
+where `T` has a schema registered (preferred), or a `PCollection<Row>`. See the
```

Comment:
I prefer `PCollection<Row>` but I think both have use cases where they are better. Can you drop the (preferred)?

Reply:
Done